### PR TITLE
Adds Multi-antag hud, which shows multple antagonists on one person

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

Multiantag hud.

Posted early because theres so much shitcode and I need help 👍 

Also fucking fixes mindslaves. Shit code incarnate.

## Why It's Good For The Game

Makes it clear when a traitor has been converted to a rev or cultist, etc

## Images of changes

image

## Testing

WIP

## Changelog
:cl:
add: Using antag-hud as a ghost will now show a list of antag statuses
fix: Nukies that use mindslaves no longer disappear from their teams
fix: Cultists that are auto-tot'd no longer appear as a non-cultist (if this ever even happens)
/:cl:

### CODE REVIEW

**Comment by DGamerL** [comment]
Reopening because it needed reviews (I'm reviewing it)

File: code/modules/antagonists/_common/antag_hud.dm [hunk 1]
```diff
@@ -56,25 +67,118 @@
...
+	return TRUE
+
+/datum/mindslaves/proc/remove_servant(datum/mind/_mind)
+	if(_mind.mindslave_slave != src)
+		return FALSE
+	_mind.remove_antag_hud(_mind.current, thrallhud, serv[_mind])
+	serv -= _mind
+	_mind.mindslave_slave = null
+	return TRUE
+
```
Comment (DGamerL) [inline code/modules/antagonists/_common/antag_hud.dm:117]:
```suggestion
```

File: code/modules/antagonists/_common/antag_hud.dm [hunk 2]
```diff
@@ -56,25 +67,118 @@
...
+/datum/antag_hud_helper/proc/transfer_antag_huds(mob/old_mob, mob/new_mob)
+	if(!istype(old_mob))
+		CRASH("transfer_antag_huds()'s old_mob: [old_mob] ([old_mob.type]) is not a mob!")
+	if(!istype(new_mob))
+		CRASH("transfer_antag_huds()'s new_mob: [new_mob] ([new_mob.type]) is not a mob!")
+	new_mob.hud_list[SPECIALROLE_HUD] = old_mob.hud_list[SPECIALROLE_HUD]
+	old_mob.hud_list[SPECIALROLE_HUD] = list()
+	var/list/temp_huds = antag_huds.Copy()
+	our_mind.leave_all_huds()
+	for(var/key as anything in new_mob.hud_list[SPECIALROLE_HUD])
```
Comment (DGamerL) [inline code/modules/antagonists/_common/antag_hud.dm:168]: Please fix CI failing on this

**Review by DGamerL — COMMENTED** [review]
Quick overview to fight the stale, is this ready to come out of draft?

**Comment by Contrabang** [comment]
This PR is buggy and I don't feel like fixing it, maybe someday in the future

**Pull request closed without merging**